index.html

```diff
@@ -51,7 +51,7 @@ <h2 class="Introduction">I am a Software Developer</h2>
         <h1>My Recent Works</h1>
         <hr class="small-line">
     </div>
-    <section class="work-section">
+    <div class="work-section">
         <div class="story">
             <img src="images/Img Placeholder.png" alt="portfolio-image">
             <div class="description-block">
```

Comment:
- [ ] Dear @Unleashedicon, One of the core requirements of this project is to have a responsive design up to the desktop width `768px`. I will go into details in bullet points:
  -  Minimum mobile screen size should be 375px. Errors in screen sizes less than 375px should be ignored. Otherwise, mobile version should be responsive until the desktop screen size.
- Previous reviewer mentioned that all your design has been compromised and it doesn't look like the figma design. Actually what happened is your design looks perfect at `375px` or lower width. but when you see it at more than `375px` then it collapses.
- Microverse urgs us  to have a responsive design in our portfolio so that visitors can view our beautiful portfolio at all screen sizes. 
- [ ] As you have not pushed the `style.css` file, so I cannot see where is the real problem in your styling. but what I am sure of that you may need to work on to make your design responsive up to `768px` which is the lowest desktop width. and also you may need to apply `display: grid` on your works section container to layout your cards.
your design now at above `375px`
![er](https://user-images.githubusercontent.com/96683843/213865038-689c40ab-d26b-450b-b42d-1d523e440de9.jpg)
![asdf](https://user-images.githubusercontent.com/96683843/213865078-355e1676-54e1-4d19-90c9-6112497e3015.jpg)


![qwer](https://user-images.githubusercontent.com/96683843/213865054-e5ee4798-2cab-48c9-9b82-7e96a89db3a8.jpg)